notebooks/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -18,7 +18,7 @@
     "### Example: SLi_Rec : Adaptive User Modeling with Long and Short-Term Preferences for Personailzed Recommendation\n",
```

Comment:
<p>Could you please add the result of the computation of NextItNet on this table?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
It seems that Amazon dataset does not have strong sequence, which is important for NextItNet. So result on Amazon dataset in this notebook is not as good as other algorigthms.
But a local test on a movielen dataset provided by author Yuan Fajie has proven the correctness of the code.
Should I write an explanation on the notebook?

Reply:
I think that we should still be transparent and show the results side by side with the other algos. At the end, they are all sequential algorithms. I agree that it would be a good idea to write an explanation on why the results look the way they look

Reply:
With the help of Jianxun, result is added to the notebook. I also added some explanation on the result